src/components/Mobile/components/FilterMobile/FilterModal.tsx

```diff
@@ -49,8 +58,11 @@ const FilterModal = (props: FilterModalProps) => {
       case "category":
```

Comment:
I remember a discussion about the enum/type for each case. if it's possible it be better.